app/src/main/java/com/github/pwittchen/reactivewifi/app/MainActivity.java

```diff
@@ -58,6 +60,12 @@
     lvAccessPoints = (ListView) findViewById(R.id.access_points);
     tvWifiSignalLevel = (TextView) findViewById(R.id.wifi_signal_level);
     tvWiFiConnection = (TextView) findViewById(R.id.wifi_connection_result);
+    showDefault();
+  }
+
+  private void showDefault() {
```

Comment:
I don't like this method name. I would rename it to something like "displayDefaultWifiConnectionState()". Name "showDefault()" is too general.